File: debug/Actors/Npc/Npc.lua
function StartAction()
   head=Actor.Animation("Actors/living/head/head.png","0,",100,200,-30,-140,1,false)
   HoldPistol=Actor.Action("Shot","engine:RemoveActor(Caller)")
   HoldPistol:AddAnimation(head)
   HoldPistol:SetMainAnimation(0)
   TBFE.TBFE_Base_MainPlayer:AddAction(HoldPistol)
end
function TalkAction()
end

File: src/tbfe/actor/Npc.cpp
#include "tbfe/actor/Npc.h"
//(PositionX, PositionY, Hair, Facial Hair, Hair Color, Torso, Frame Lengths, Frames,
//Number of Frames, Number Of Frame Sets)
Npc::Npc():Actor(0,0)
{
};
Npc::Npc(int PositionX,int PositionY,string PlayerConversation)
:Actor(PositionX,PositionY)
{
  setScriptSource("Actors/Npc/Npc.py");
  string StandAnimation="0,";
  Action Stand("Stand","");
  Animation Body("test.dae",StandAnimation,
		 0,0,1,true);
  Quaternion rotation=Quaternion(1,0,0,270);
  Body.setRotation(rotation);
  Stand.addAnimation(Body);
  Stand.setMainAnimation(0);
  addAction(Stand);
};
Npc::~Npc()
{
};
